SettingsList.py

```diff
@@ -2467,6 +2461,39 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             <b>Spirit Temple</b>: lobby elevator activated,
             shortcut silver blocks moved, central room
             platform lowered, and statue face melted
+
+            Choice : Select dungeons with shortcuts
+            All : Enable all dungeons shortcuts
+            Random : Random dungeon shortcuts
```

Comment:
Sorry forgot this on my first review, one last spacing nitpick: Should be no space directly before the colon.

Reply:
Fix https://github.com/TestRunnerSRL/OoT-Randomizer/pull/1493/commits/b3958c886644d5d247a9adc47cb33b4e9a50bafb